vxquery-xtest/results/xqts.txt

```diff
@@ -14370,30 +14370,30 @@ Functions/NodeFunc/NodeRootFunc//K2-NodeRootFunc-6, EXPECTED_RESULT_GOT_ERROR
 Functions/NodeFunc/NodeRootFunc//K2-NodeRootFunc-7, EXPECTED_RESULT_GOT_ERROR
 Functions/NodeFunc/NodeRootFunc//K2-NodeRootFunc-8, EXPECTED_RESULT_GOT_ERROR
 Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-1, EXPECTED_ERROR_GOT_SAME_ERROR
-Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-10, EXPECTED_RESULT_GOT_ERROR
-Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-10d, EXPECTED_RESULT_GOT_FAILURE
-Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-2, EXPECTED_ERROR_GOT_FAILURE
-Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-3, EXPECTED_ERROR_GOT_FAILURE
+Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-10, EXPECTED_RESULT_GOT_FAILURE
+Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-10d, EXPECTED_RESULT_GOT_ERROR
+Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-2, EXPECTED_ERROR_GOT_RESULT
+Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-3, EXPECTED_ERROR_GOT_RESULT
 Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-4, EXPECTED_RESULT_GOT_ERROR
 Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-4d, EXPECTED_RESULT_GOT_ERROR
 Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-5, EXPECTED_RESULT_GOT_ERROR
 Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-5d, EXPECTED_RESULT_GOT_ERROR
-Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-6, EXPECTED_RESULT_GOT_ERROR
-Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-7, EXPECTED_ERROR_GOT_DIFFERENT_ERROR
-Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-8, EXPECTED_RESULT_GOT_ERROR
-Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-9, EXPECTED_RESULT_GOT_FAILURE
+Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-6, EXPECTED_RESULT_GOT_FAILURE
+Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-7, EXPECTED_ERROR_GOT_FAILURE
+Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-8, EXPECTED_RESULT_GOT_FAILURE
+Functions/NodeSeqFunc/SeqCollectionFunc//fn-collection-9, EXPECTED_RESULT_GOT_ERROR
 Functions/NodeSeqFunc/SeqDocAvailableFunc//fn-doc-available-1, EXPECTED_ERROR_GOT_SAME_ERROR
 Functions/NodeSeqFunc/SeqDocAvailableFunc//fn-doc-available-2, EXPECTED_ERROR_GOT_RESULT
 Functions/NodeSeqFunc/SeqDocAvailableFunc//fn-doc-available-3, EXPECTED_RESULT_GOT_FAILURE
 Functions/NodeSeqFunc/SeqDocAvailableFunc//fn-doc-available-4, EXPECTED_RESULT_GOT_FAILURE
 Functions/NodeSeqFunc/SeqDocAvailableFunc//fn-doc-available-5, EXPECTED_RESULT_GOT_ERROR
-Functions/NodeSeqFunc/SeqDocAvailableFunc//fn-doc-available-6, EXPECTED_RESULT_GOT_SAME_RESULT
+Functions/NodeSeqFunc/SeqDocAvailableFunc//fn-doc-available-6, EXPECTED_RESULT_GOT_DIFFERENT_RESULT
```

Comment:
This is fixed now. The problem was that even if the document was not available the doc_available function returned true.